### DISCUSSION TITLE
一些自己做的小游戏

### DISCUSSION THREAD

**Discussion by qiufengcute** [share121/inter-knot#1701] [其他]
这是一些我自己做的小游戏：https://nsx.lanzout.com/b00jdl36wb 密码：epe0

**Comment by Coqwq** [comment]
我的电脑觉得不安全，玩不了QAQ

**Comment by Tkingxiao** [comment]
在wordgame，打了三次野兽然后跳没了！！！但是！不错的游戏！比肩 黑神话
